php_apcu/README.md

```diff
@@ -0,0 +1,81 @@
+# Agent Check: php_apcu
+
+## Overview
+
+This check monitors [PHP APCu][1] through the Datadog Agent.
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying these instructions.
+
+### Installation
+
+To install the `php_apcu` check on your host:
+
+
+1. Install the [developer toolkit][3].
+ on any machine.
+
+2. Run `ddev release build php_apcu` to build the package.
+
+3. [Download the Datadog Agent][4].
+
+4. Upload the build artifact to any host with an Agent and
+ run `datadog-agent integration install -w
+ path/to/php_apcu/dist/<ARTIFACT_NAME>.whl`.
+
+#### APCu Setup
+
+Provide a URL to the collection program.
+
+APCu does not have a metric output by itself.
+
+* /opt/datadog-agent/embedded/lib/python3.8/site-packages/datadog_checks/php_apcu/assets/exporter/apcu-dd-handler.php
+
+It will work if you refer to it as a php file, but you may want to configure it as follows
+
+```
+Alias /apcu-status /opt/datadog-agent/embedded/lib/python3.8/site-packages/datadog_checks/php_apcu/assets/exporter/apcu-dd-handler.php
+<Location /apcu-status>
+    Require all denied
+    Require local
+</Location>
+```
```

Comment:
Where is this part meant to be configured, is there a file or is this a command to run? In this path `/opt/datadog-agent/embedded/lib/python3.8/site-packages/datadog_checks/php_apcu/assets/exporter/apcu-dd-handler.php`?

If so let's reword this line to be more specific:
```
It will work if you refer to it as a php file, but you may want to configure it as follows
```

Reply:
The user needs to write it in the configuration file of apache.
To be precise, it needs to be written in the configuration file of the web server that the user is using.
However, since php does not depend on the web server, I am using the well-known Apache example.

The mod_status of apache also needs to be set, but it was just one line.
https://github.com/DataDog/integrations-core/tree/master/apache#installation
````
Install mod_status on your Apache servers and enable ExtendedStatus.
```

https://github.com/DataDog/integrations-core/tree/master/nginx#prepare-nginx
I wrote this in the README based on the nginx example.

>It will work if you refer to it as a php file, but you may want to configure it as follows
This file must be run through a web server, so writing it that way may cause confusion.

Reply:
Oooooh, okay. Thank you for this clarification. I am going to re-edit.